contracts/Oracles/ChainlinkPriceOracleWstETH.sol

```diff
@@ -1,29 +1,34 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.19;
 
+import { Fixed256x18 } from "@tempusfinance/tempus-utils/contracts/math/Fixed256x18.sol";
 import { IWstETH } from "../Dependencies/IWstETH.sol";
-import { AggregatorV3Interface, BaseChainlinkPriceOracle } from "./BaseChainlinkPriceOracle.sol";
-import { BasePriceOracleWstETH } from "./BasePriceOracleWstETH.sol";
+import { AggregatorV3Interface, ChainlinkPriceOracle } from "./ChainlinkPriceOracle.sol";
+import { IPriceOracleWstETH } from "./Interfaces/IPriceOracleWstETH.sol";
 
-contract ChainlinkPriceOracleWstETH is BaseChainlinkPriceOracle, BasePriceOracleWstETH {
-    // --- Constants ---
+contract ChainlinkPriceOracleWstETH is IPriceOracleWstETH, ChainlinkPriceOracle {
+    using Fixed256x18 for uint256;
 
-    uint256 public constant override DEVIATION = 1e16; // 1%
+    // --- Immutables ---
+
+    IWstETH public immutable override wstETH;
```

Comment:
This part is now duplicated, and earlier, it wasn't (there was an abstract class for this)